pkg/operator/apis/config/v1alpha1/defaults.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	componentbaseconfigv1alpha1 "k8s.io/component-base/config/v1alpha1"
+
+	"github.com/gardener/gardener/pkg/logger"
+)
+
+func addDefaultingFuncs(scheme *runtime.Scheme) error {
+	return RegisterDefaults(scheme)
+}
+
+// SetDefaults_OperatorConfiguration sets defaults for the configuration of the Gardener operator.
+func SetDefaults_OperatorConfiguration(obj *OperatorConfiguration) {
+	if obj.LogLevel == "" {
+		obj.LogLevel = logger.InfoLevel
+	}
+	if obj.LogFormat == "" {
+		obj.LogFormat = logger.FormatJSON
+	}
+}
+
+// SetDefaults_ClientConnectionConfiguration sets defaults for the garden client connection.
+func SetDefaults_ClientConnectionConfiguration(obj *componentbaseconfigv1alpha1.ClientConnectionConfiguration) {
+	if obj.QPS == 0.0 {
+		obj.QPS = 50.0
```

Comment:
```suggestion
                obj.QPS = 100.0
```
To make it consistent with `example/operator/10-componentconfig.yaml`